common_features.mk

```
@@ -345,6 +370,10 @@ ifeq ($(strip $(WS2812_DRIVER_REQUIRED)), yes)
     endif
 endif
 
+ifeq ($(strip $(APA102_DRIVER_REQUIRED)), yes)
+    SRC += apa102.c
+endif
+
```

```suggestion
ifeq ($(strip $(APA102_DRIVER_REQUIRED)), yes)
    COMMON_VPATH += $(DRIVER_PATH)/apa102
    SRC += apa102.c
endif
```

I agree it shouldn't live in quantum. I just put them there for testing because there was already a variable for that directory :)

Moving it to `drivers/apa102/apa102.[c|h]` would make sense imo, necessitating this path addition :)